skaha/src/main/java/org/opencadc/skaha/session/PostAction.java

```diff
@@ -978,5 +971,20 @@ private String getGPUScheduling(Integer gpus) {
         }
         return sb.toString();
     }
+    
+    private String getHeadlessPriority() {
+        if (skahaPriorityHeadlessGroup == null) {
+            return "";
+        }
+        if (skahaHeadlessPriortyClass == null) {
+            log.warn("headlessPriorityGroup set but headlessPriorityClass not set");
```

Comment:
Should this actually throw an Exception or is this an acceptable incomplete configuration?

Reply:
I figure that if deployers don't need this headless priority function then they shouldn't need to configure it.  So, left it as just a warning.